gpt_index/embeddings/openai.py

```diff
@@ -3,7 +3,8 @@
 from enum import Enum
 from typing import List
 
-from openai.embeddings_utils import get_embedding
+import openai
+from tenacity import retry, stop_after_attempt, wait_random_exponential
```

Comment:
I don't know if you actually want to keep all the retry logic, but that's what they have in original code.